libraries/AP_InertialSensor/AP_InertialSensor_MPU6000.h

```diff
@@ -18,7 +18,11 @@
 #define MPU6000_DEBUG 0
 
 #define MPU6000_SAMPLE_SIZE 14
+#if CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_BH
+#define MPU6000_MAX_FIFO_SAMPLES 84
```

Comment:
why 84 samples???  This would actually give pretty bad results. We can't let that many samples to pile up. Bebop also uses MPU6050  (I2C) and it's the one MPU6000_MAX_FIFO_SAMPLES is defined for.

@jberaud any comments here?